code/modules/mob/living/simple_animal/constructs/constructs.dm

```diff
@@ -233,6 +237,7 @@
 	resistance = 10
 	var/energy = 0
 	var/max_energy = 1000
+	can_climb = 1 //It flies, its magic
```

Comment:
and if they cant climb ladders, how else should they get around the station?

Nonhumanoids also can't operate elevators afaik, and they're far too large for ventcrawling